src/Components/BatchesList.test.tsx

```diff
@@ -0,0 +1,310 @@
+            error_info: ""
+        },
+        {
+            batch: "OPN_12032021_023400",
+            dd_filename: "dd_OPN2101A_26032021_121540.zip",
+            instrumentName: "OPN2101A",
+            state: "generated",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        },
+        {
+            batch: "OPN_12032021_023400",
+            dd_filename: "dd_OPN2101A_26032021_121540.zip",
+            instrumentName: "OPN2101A",
+            state: "inactive",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        }
+    ];
+
+    const pendingBatchInfoList: DataDeliveryFileStatus[] = [
+        {
+            batch: "LM_12032021_023398",
+            dd_filename: "LM2004A",
+            instrumentName: "LM2004A",
+            state: "in_arc",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        },
+        {
+            batch: "LM_12032021_023398",
+            dd_filename: "dd_LM2101A_26032021_121540.zip",
+            instrumentName: "LM2101A",
+            state: "in_arc",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        },
+        {
+            batch: "LM_12032021_023398",
+            dd_filename: "dd_LM2101A_26032021_121540.zip",
+            instrumentName: "LM2101A",
+            state: "chicken",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        }
+    ];
+
+    const successBatchInfoList: DataDeliveryFileStatus[] = [
+        {
+            batch: "LM_12032021_876000",
+            dd_filename: "LM2004A",
+            instrumentName: "LM2004A",
+            state: "in_arc",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        },
+        {
+            batch: "LM_12032021_876000",
+            dd_filename: "dd_LM2101A_26032021_121540.zip",
+            instrumentName: "LM2101A",
+            state: "in_arc",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        },
+        {
+            batch: "LM_12032021_876000",
+            dd_filename: "dd_LM2101A_26032021_121540.zip",
+            instrumentName: "LM2101A",
+            state: "in_arc",
+            updated_at: "2021-03-26T12:21:10+00:00",
+            error_info: ""
+        }
+    ];
+
+    afterEach(() => {
+        jest.clearAllMocks();
+        cleanup();
+    });
+
+    it("displays a red circle when a batch entry has errored ", async () => {
+        mock.onGet("/api/batch").reply(200, [batches[0]]);
+        mock.onGet("/api/batch/OPN_24032021_113000").reply(200, errorBatchInfoList);
+        
+        const history = createMemoryHistory();
+        const wrapper = render(
+            <Router history={history}>
+                <BatchesList/>
+            </Router>
+        );
+
+        expect(screen.queryByText(/Loading/i)).toBeInTheDocument();
```

Comment:
This could be written with out the `waitFor`:

```typescript
expect(screen.queryByText(/Loading/i)).toBeInTheDocument();
await expect(screen.findByTestId(/OPN_24032021_113000-status-error/i)).toBeDefined();
expect(screen.queryByText(/Loading/i)).not.toBeInTheDocument();
```